### DISCUSSION TITLE
跨章节引用失败

### DISCUSSION THREAD

**Discussion by Bennett561** [sjtug/SJTUThesis#738] [提问] [answered]
跨章节是无法引用吗？我的第三章和第四章是分开两个tex文件，想在第四章中引用第三章的subsection编号，用compile.bat编译出来是"??"...

**Comment by skyzh** [comment, marked as answer]
Hi, 请提供相关代码，<del>以及你的编译方法。</del>

如果是 Compile.bat，怀疑你的引用方法有误。是否给 subsection 打上了 label？

**Reply by Bennett561** [reply]
在第三章中
\subsection{xx} \label{sec:xx}

在第四章中
\ref{sec:xx}

使用compile.bat编译

**Reply by Bennett561** [reply]
哎，我今天啥也没改，重新编译一次他又正常了！
